deploy/charts/rook-ceph/values.yaml

```diff
@@ -52,6 +52,10 @@ rbacEnable: true
 ##
 pspEnable: true
 
+monitoring:
+  # enabling will create RBAC rules to allow Operator to create ServiceMonitors
+  enabled: true
+
```

Comment:
Monitoring should not be enabled for all users by default. For our releases that go out to hundreds of users, we'd like this to be false by default, and for those who want monitoring, they enable it via `monitoring.enabled`.